robolectric/src/main/java/org/robolectric/internal/bytecode/InstrumentationConfiguration.java

```diff
@@ -187,22 +209,13 @@ public boolean shouldAcquire(String name) {
     // See https://github.com/robolectric/robolectric/issues/521
     if (name.equals("android.R$styleable")) return true;
 
-    return !(
-        name.matches(".*\\.R(|\\$[a-z]+)$")
-            || classesToNotAquire.contains(name)
-            || name.startsWith("java.")
-            || name.startsWith("javax.")
-            || name.startsWith("sun.")
-            || name.startsWith("com.sun.")
-            || name.startsWith("org.w3c.")
-            || name.startsWith("org.xml.")
-            || name.startsWith("org.junit")
-            || name.startsWith("org.hamcrest")
-            || name.startsWith("org.specs2")  // allows for android projects with mixed scala\java tests to be
-            || name.startsWith("scala.")      //  run with Maven Surefire (see the RoboSpecs project on github)
-            || name.startsWith("kotlin.")
-            || name.startsWith("com.almworks.sqlite4java") // Fix #958: SQLite native library must be loaded once.
-    );
+    if (packagesToNotAquire.contains(pkgName)) return false;
```

Comment:
Also there is one difference. Current implementation ensures that we check whole packages and not just part. If we change to `startsWith` then adding 'org.scala' to `packagesToNotAquire` will exclude several packages, for example: `org.scalatest', `org.scalactic`, etc.
In some cases this could be desired behaviour, but sometimes could be confusing. 
Of course we can prevent that with something like that: `packageName.startsWith(packageToNotAquire + ".")`
Let me know which version you prefer.